File: c/Estruturas repetitivas/senha.c
#include <stdio.h>

int main() {
	int senhaCorreta, senha;

	senhaCorreta = 2002;

	printf("Digite sua senha:\n");
	scanf("%d", &senha);

	while(senha != senhaCorreta) {
		printf("Senha incorreta\n");
		printf("Digite sua senha:\n");
		scanf("%d", &senha);
	}

	if (senha == senhaCorreta);
	{
		printf("Acesso liberado");
	}
	
		return 0;

}

File: java/Estruturas repetitivas/Senha.java
import java.util.Scanner;

public class Senha {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int senha;

        System.out.print("Digite a senha: ");
        senha = sc.nextInt();

        while (senha != 2002) {
            System.out.println("Senha Invalida! Tente novamente:");

            System.out.print("Digite a senha: ");
            senha = sc.nextInt();
        }

        System.out.println("Acesso Permitido");

        sc.close();
    }
}